crashsimilarity/models/doc2vec.py

```diff
@@ -38,13 +38,13 @@ def _train_model(self, force_train=False):
         except NotImplementedError:
             workers = 2
 
-        model = gensim.models.Doc2Vec(size=100, window=8, iter=20, workers=workers)
+        model = gensim.models.Doc2Vec(vector_size=100, window=8, epochs=20, workers=workers)
         model.build_vocab(self._corpus)
         logging.debug("Vocab Length{}".format(len(model.wv.vocab)))
 
         t = time.time()
         logging.info('Training model...')
-        model.train(self._corpus)
+        model.train(self._corpus, total_examples=model.corpus_count, epochs=model.epochs)
```

Comment:
Are these parameters needed?

Reply:
Yes, they are not optional in the newer versions of gensim